pyproject.toml

```diff
@@ -198,6 +198,8 @@ filterwarnings = [
     "ignore:'cgi' is deprecated and slated for removal in Python 3.13:DeprecationWarning",
     # Ignore numpy 2.0 warning, see https://github.com/astropy/astropy/pull/15495
     "ignore:numpy.core._multiarray_umath is deprecated:DeprecationWarning",
```

Comment:
Note that I tried to add `pyerfa` here at the end but that does not work. This may be related to our own build system, so out of scope here.